letsdrill.rb

```diff
@@ -1,13 +1,37 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  if integer > 100 
+    "A++"
+  elsif integer >=90
+     "A"
+  elsif integer >=80 
+    "B"
+  elsif integer >=70
+    "C"
+  elsif integer >=60
+    "D"
+  elsif integer >=50
+     "E"
+  else
+    "F"
+  end
 
 end
 
 def shortest_string(array)
+  shortest_index = 0
+  index = 0
 
-  #Put your code here!
+  for str in array
```

Comment:
we don't use loops in ruby. It is a feature of ruby, but no one does it, because using enumerable#each is considered more idiomatic.